cmd/sriovdp/main.go

```diff
@@ -43,7 +46,7 @@ func main() {
 
 	glog.Infof("resource manager reading configs")
 	if err := rm.readConfig(); err != nil {
-		glog.Errorf("error getting resources from file %v", err)
+		glog.Error("error getting resources from file %v", err)
```

Comment:
nit: I think `glog.Errorf(...)` was good, as it contains a `%v`

Reply:
fixed

Reply:
was not fixed. its still glog.Error.